packages/accounts/src/light-account/__tests__/account.test.ts

```diff
@@ -92,7 +94,7 @@ const givenConnectedProvider = ({
       entryPointAddress: dummyEntryPointAddress,
       chain,
       owner,
-      factoryAddress: "0xLIGHT_ACCOUNT_FACTORY_ADDRESS",
+      factoryAddress: dummyFactoryAddress,
```

Comment:
Like I do in pr https://github.com/alchemyplatform/aa-sdk/pull/181, we should just use default light account utils for our tests as well. 

I think this pr updates all instances with using the default helper. Might want to rebase development after https://github.com/alchemyplatform/aa-sdk/pull/181 is merged?

Reply:
yup totally, let's get 181 in and I'll rebase